File: trail1/3 또는 5의 배수/multiples-of-3-or-5.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        String val1 = (A % 3 == 0) ? "YES" : "NO";
        String val2 = (A % 5 == 0) ? "YES" : "NO";
        System.out.println(val1 + "\n" + val2);
    }
}

File: trail1/3 또는 5의 배수/multiples-of-3-or-5.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int A;
    cin >> A;
    string val1 = A%3==0 ? "YES" : "NO";
    string val2 = A%5==0 ? "YES" : "NO";
    cout << val1 << "\n" << val2 << "\n";
    return 0;
}